tensorflow/lite/micro/kernels/add_common.cc

```diff
@@ -39,6 +39,10 @@ TfLiteStatus CalculateOpDataAdd(TfLiteContext* context, TfLiteAddParams* params,
   data->requires_broadcast = !HaveSameShapes(input1, input2);
 
   if (output->type == kTfLiteInt8 || output->type == kTfLiteInt16) {
+    // This is to ensure that for int8/int16 output type,output does not have
```

Comment:
Nit: Remove the comment. It's basically explaining the code.

Reply:
Done.